.github/workflows/release.yml

```diff
@@ -0,0 +1,41 @@
+name: publish
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  pypi_packages:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch_depth: 0
+
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
+
+      - name: build
+        run: |
+          python -m pip install wheel twine
+          python -m pip install -r requirements.txt
+          python setup.py sdist
+          pip wheel . -w dist --no-deps
+      - name: Check files
+        run: |
+          ls dist
+        shell: bash
+
+      - name: Test wheels
+        run: |
+          python -m twine check dist/*
+        shell: bash
+
+      - name: Publish a Python distribution to PyPI
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_PASSWORD }}
```

Comment:
Did you add the secret to the repo or do you want me to do so?

Reply:
I've added it already